llvm/tools/bugpoint/BugDriver.h

```diff
@@ -183,7 +183,7 @@ class BugDriver {
   /// MayModifySemantics argument is true, then the cleanups is allowed to
   /// modify how the code behaves.
   ///
-  std::unique_ptr<Module> performFinalCleanups(std::unique_ptr<Module> M,
+  std::unique_ptr<Module> performFinalCleanups(const std::unique_ptr<Module> &M,
```

Comment:
Passing unique_ptr by value or const reference are both suspect. I would expect this to take const Module&